src/handle_provide_parameter.c

```diff
@@ -84,15 +84,15 @@ static void handle_claim_multisig_account(ethPluginProvideParameter_t *msg, cont
 static void handle_reward_create_position(ethPluginProvideParameter_t *msg, context_t *context) {
     switch (context->next_param) {
         case AMOUNT:  // amount
-            copy_parameter(context->lisk.body.rewardCreatePosition.lock_amount,
+            copy_parameter(context->lisk.body.rewardCreatePosition.first,
                            msg->parameter,
-                           sizeof(context->lisk.body.rewardCreatePosition.lock_amount));
+                           sizeof(context->lisk.body.rewardCreatePosition.first));
             context->next_param = DURATION;
             break;
         case DURATION:  // lockingDuration
-            copy_parameter(context->lisk.body.rewardCreatePosition.lock_duration,
+            copy_parameter(context->lisk.body.rewardCreatePosition.secound,
                            msg->parameter,
-                           sizeof(context->lisk.body.rewardCreatePosition.lock_duration));
+                           sizeof(context->lisk.body.rewardCreatePosition.secound));
```

Comment:
Just a typo, please update at all places
```suggestion
                           sizeof(context->lisk.body.rewardCreatePosition.secound));
```
```suggestion
                           sizeof(context->lisk.body.rewardCreatePosition.second));
```